handlers/slog/options.go

```diff
@@ -0,0 +1,50 @@
+package sloghandler
+
+import (
+	"io"
+	"log/slog"
+
+	"github.com/vpakhuchyi/censor"
+)
+
+// Option represents a set of options for configuring the Slog Handler along with a censor processor.
+// It is used as arguments for the NewJSONHandler function. See NewJSONHandler for more details.
+type Option func(options *config)
```

Comment:
I think the current naming in this signature may confuse a bit. 
`type Option func(options *config)` - so i expect to see a func that accepts options but it accepts a config.
What do you think about changing it to `type Option func(cfg *config)`?

Reply:
Make sense, thank you